compiler/parse.js

```diff
@@ -17,7 +17,7 @@ globalThis.parser = '';
 let parse;
 const loadParser = async (fallbackParser = 'acorn', forceParser) => {
   parser = forceParser ?? Prefs.parser ?? fallbackParser;
-  const mod = (await import((globalThis.document || globalThis.Deno ? 'https://esm.sh/' : '') + parser));
+  const mod = (await import((globalThis.document ? 'https://esm.sh/' : '') + parser));
```

Comment:
I think this will break parsers which aren't `acorn` (used for typescript etc)?

Reply:
fwiw, I've tested all of the parsers and they do seem to work on the latest Deno